tests/replicaset/manager-getservers-001.phpt

```diff
@@ -76,7 +76,7 @@ array(3) {
       ["primary"]=>
       string(19) "192.168.112.10:3000"
       ["tags"]=>
-      object(stdClass)#%d (%d) {
+      array(2) {
```

Comment:
This is the result of `php_phongo_server_to_zval()` using `state.map.document_type = PHONGO_TYPEMAP_NATIVE_ARRAY;` when decoding `last_is_master`, and that mapping applying both to top-level and embedded documents alike.

Same concern that you raised in https://github.com/10gen-labs/mongo-php-driver-prototype/pull/56/files#r34193151. In order to fix this, I believe [PHPC-319](https://jira.mongodb.org/browse/PHPC-319) should depend on [PHPC-315](https://jira.mongodb.org/browse/PHPC-315) and I need to create the "root" mapping first.